### PR TITLE
Typography examples

### PR DESCRIPTION
**Pull request by pauginer**
Added Do’s and Don’ts for contrast and an example for the dynamic text
concept. Also added links about typefaces when introduced.

### CODE REVIEW

File: visual-style_typography.html [hunk 1]
```diff
@@ -92,6 +98,11 @@ <h1 class="page__title">Visual Style</h1>
 						<h4>Dynamic text</h4>
 						<p>Content will be available in multiple languages, so text length will vary for a given piece of information across languages. Avoid designing interfaces that depend on certain expectation of text length.</p>
 
+						<figure>
+							<img src="img/visual-style/typography-dynamic-text.png" alt="Text shown at differnt font sizes depending on their length.">
```
Comment (Volker-E) [inline visual-style_typography.html]: typo: >"different". We also don't use punctuation in alt attributes' contents

Comment (Volker-E) [inline visual-style_typography.html]: Is the "Don't" style accorded? Also don't we go for "Do"?

Comment (pauginer) [inline visual-style_typography.html]: For the style of Do's and Dont's we still need to define a general template. We can iterate on this as we integrate the images in the Sketch file.

**Comment by Volker-E** [comment]
For notice: So far we've been putting in @2x pixel images, which are scaled down if needed, to also provide basic support for HiDPI devices. 
I won't block this PR as we are going to have the automatic export in various formats including device-independent SVGs asap.

**Pull request merged**